pymc3/__init__.py

```diff
@@ -13,7 +13,7 @@
 #   limitations under the License.
 
 # pylint: disable=wildcard-import
-__version__ = "3.11.2"
+__version__ = "3.11.3"
```

Comment:
Hmm, where did this come from? Some artifact from me having changed the base branch?

Reply:
Now it's fixed after a rebase.